src/AbstractErrorHandler.php

```diff
@@ -150,16 +150,15 @@ public function captureAt($levels, $replace = false)
      * @param string $file    The filename the error was raised in
      * @param int    $line    The line number the error was raised at
      *
-     * @return bool Whether the standard PHP error handler should be called
+     * @return bool If the function returns FALSE then the normal error handler continues
      *
      * @internal
      */
     public function handleError($level, $message, $file, $line)
```

Comment:
Please review thoroughly the changes to this methods, since it was severely bugged and those are changing the behavior.